truss/templates/trtllm/packages/constants.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
+
+# If changing model repo path, please updated inside tensorrt_llm config.pbtxt as well
```

Comment:
Can we make a note to parametrize and render these pbtxt at truss load time as with a pattern like in the TRT-LLM backend [tooling](https://github.com/triton-inference-server/tensorrtllm_backend/blob/7982723f13ddad4a478093fea02f55441b3608b0/tools/fill_template.py)?